configs/basic.js

```diff
@@ -1,156 +1,13 @@
 module.exports = {
-  extends: [
-    'tslint:all'
-  ],
-
   rules: {
-    align: [
-      true,
-      'parameters',
-      'statements'
-    ],
     'array-type': [
       true,
       'array'
```

Comment:
off-topic:

can someone explain to me why one is preferred over the other?
https://palantir.github.io/tslint/rules/array-type/
i'm confused by the array-simple option, for instance... makes it sound like there's a problem with `Array<T>` if T is not simple

Reply:
I interpret it is this way:
Use T[] if T is a "simple" type, e.g.:
```
  foo: number[];
  bar: User[];
```

but for a non-simple type you must use `Array` unless you first declare a type for the non-simpe type, e.g.:
```
  foo: Array<number | string | User>
```

would be ok with `array-simple` but with `array` you must do:

Reply:
jaha, maybe. I read simple as primitive/built-in not "declared"
your guess makes sense, making "array" as a way to enforce declaration of types, which i see as a benefit

Reply:
Agreed.